File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/direct/comment/uncomment/direct/Java/Go/s063482767_commented.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
	"strconv"
	"strings"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	n, _ := strconv.Atoi(scanner.Text())
	nums := make([]int, n)
	scanner.Scan()
	input := strings.Fields(scanner.Text())
	for i := range nums {
		nums[i], _ = strconv.Atoi(input[i])
	}
	sort.Ints(nums)
	sum := 0
	for i := 0; i < n-1; i++ {
		sum += nums[i]
	}
	if nums[n-1] < sum {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/comment/direct/Java/Python/s063482767_commented.py
import heapq

n = int(input())
a = list(map(int, input().split()))
heapq.heapify(a)
sum = 0
for _ in range(n-1):
    x = heapq.heappop(a)
    sum += x
last = heapq.heappop(a)
print("Yes" if last < sum else "No")
# End of Code
